src/hooks/loginHooks/useSgidLogin.ts

```diff
@@ -3,17 +3,21 @@ import { useQuery, UseQueryResult } from "react-query"
 
 import * as LoginService from "services/LoginService"
 
-import { VerifySgidLoginParams } from "types/login"
+import { PublicOfficerData, VerifySgidLoginParams } from "types/login"
 
 export const useSgidLogin = (
   body: VerifySgidLoginParams
-): UseQueryResult<void> => {
-  return useQuery([body], () => LoginService.verifySgidLogin(body), {
-    onSuccess: () => {
-      window.location.replace("/sites")
-    },
-    onError: (err: AxiosError) => {
-      window.location.replace(`/ogp-login?status=${err.response?.status}`)
-    },
-  })
+): UseQueryResult<PublicOfficerData[]> => {
```

Comment:
Shapes returned are the same (always an array of userData objects), just that if only a single item exists in the array, the frontend ignores all of it because we can log the user in directly